SampleApp/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,10 @@
+#Mon Jan 20 13:25:50 IST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+<<<<<<< HEAD
+=======
```

Comment:
This line has to be removed after resolving conflicts. It is to denote conflict.